### DISCUSSION TITLE
Image frame with no laser burn

### DISCUSSION THREAD

**Discussion by emonroe11** [arkypita/LaserGRBL#1712] [Q&A] [answered]
Hello all, Is it possible to an image frame with NO laser burn. I tried to adjust the button options but it still burns a frame.
![image](https://user-images.githubusercontent.com/97243166/148411650-6ddf0cac-a9e4-4765-9478-cbaada37f18a.png)

**Comment by emonroe11** [comment, marked as answer]
Answered my own question. Figured it out.

**Reply by Katrinka725** [reply]
Could you tell me how you corrected this. I’m new to laser engraving and am using lasergrbl and when I frame it’s engraving a thin line. I have settings turned down but it’s still doing it.

**Reply by Tribo-2dot0** [reply]
Clique com o direito sobre o botão Frame > Editar Botão, diminua o valor de S na segunda linha (imagem acima).
Podes deixar a segunda linha assim: M3 S5 F1000
Se o laser não acender aumente para S6... S7... etc.
Se quiser que faça a moldura mais rápido aumente o valor de F.
